src/libraries/Microsoft.PowerFx.Core/Public/Values/CustomObjectValue.cs

```diff
@@ -13,15 +13,7 @@ namespace Microsoft.PowerFx.Core.Public.Values
 {
     public interface ICustomObject
```

Comment:
>ICustomObject [](http://example.com/codeflow?start=21&length=13)

This interface needs some more comments.

Reply:
Added a few comments